Formula/akamai.rb

```diff
@@ -0,0 +1,27 @@
+class Akamai < Formula
+  desc "CLI toolkit for working with Akamai's API from the command-line."
+  homepage "https://github.com/akamai/cli"
+  url "https://github.com/akamai/cli.git",
+    :tag => "0.1.0"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+
+    srcpath = buildpath/"src/github.com/akamai/cli"
+    srcpath.install buildpath.children
+
+    cd srcpath do
+      system "glide", "install"
+      system "go", "build", "-o", "akamai", "."
+      bin.install "akamai"
+    end
+  end
+
+  test do
+    system "#{bin}/akamai", "--version"
```

Comment:
This will need to be a test that exercises and verifies the functionality of the software.